fiftyone/utils/cvat.py

```diff
@@ -811,11 +811,10 @@ def from_labels_dict(cls, d):
             attributes = _ensure_list(_tmp.get("attribute", []))
             _attributes = []
             for attribute in attributes:
+                _values = attribute.get("values", None)
+                _categories = _values.split("\n") if _values else []
```

Comment:
`"".split("\n")` is `[""]`, so I updated this so that missing values will result in `[]` instead, which is more reasonable I think.

This `CVATTaskLabels` is just the annotation schema, it doesn't affect things much. But, nonetheless, cleaner I think.

Reply:
Good call!